modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/dataset_display/entry.js

```diff
@@ -20,12 +20,17 @@ import {initializeConfig} from './config';
 import ViewsContext, {viewsReducer} from './views/ViewsContext';
 
 const App = ({apiURL, appURL, ...props}) => {
-	const {activeViewSettings, portletId, views} = props;
-	const activeView = activeViewSettings.name
-		? views.find(({name}) => name === activeViewSettings.name)
+	const {
+		activeViewSettings: {name: activeViewName, ...activeViewSettings},
```

Comment:
Tricky! I hadn't thought of re-using the LHS name (`activeViewSettings`) in the RHS spread.